src/s1reader/s1_etad.py

```diff
@@ -0,0 +1,202 @@
+"""Wrapper of s1etad module to read S1-ETAD products."""
+import datetime
+import glob
+import os
+
+import numpy as np
+from matplotlib import pyplot as plt
+from scipy.interpolate import RectBivariateSpline
+
+try:
+    import s1etad
+except ImportError:
+    raise ImportError('Can NOT import s1etad (https://gitlab.com/s1-etad/s1-etad)!')
+
+
+
+def get_eta_correction_from_slc_burst(slc_burst, eta_dir, corr_type='sum', include_tropo=True,
```

Comment:
Does the title of function miss a D for `eta`? Through out the PR you use ETA and ETAD but I am not sure if this is done intentionally or not

Reply:
Both ETA and ETAD are used by ESA. I am using `eta` in all func/var names, and ETAD in the comments. Changing all to `etad` sounds good to me too, let me know please.

Reply:
Your nomenclature is fine as it is. :)